### DISCUSSION TITLE
Filebeat - Data.Path Problem

### DISCUSSION THREAD

**Discussion by SpiderNet-Villach** [Security-Onion-Solutions/securityonion#3443] [Unsupported Versions]
I'm a Security Onion newbie and I'm having problems pushing logs from Wazuh (Wazuh works fine btw) to Kibana. If I'm not mistaken, Filebeat is involved in this process and since I cannot get my SO to show any logs in Kibana, I started error hunting. 

![image](https://user-images.githubusercontent.com/75905540/111171518-4ba2f180-85a5-11eb-850b-bd3e072dbe79.png)

Filebeat has connectivity to Logstash

[root@so-filebeat filebeat]# ./filebeat -c filebeat.yml -e
2021-03-15T14:55:43.577Z        ERROR   instance/beat.go:956    Exiting: data path already locked by another beat. Please make sure that multiple beats are not sharing the same data path (path.data).
Exiting: data path already locked by another beat. Please make sure that multiple beats are not sharing the same data path (path.data).

However, it seems like it does not run at all

Apparently, I could fix this error If I was able to change my filebeat.yml file - But changing this file directly without using Salt does not work. 

The problem that I'm facing now is that I'm unable to change this config-file since everything I have tried so far with Salt did not work and I cannot find any useful documentation or guides which would be helpful in solving this problem.  

Any help would be kindly appreciated!

**Comment by weslambert** [comment]
Have you tried rebooting?  It seems like it thinks a Beat is already running, so there may be some pidfile or something else that has not been properly cleaned up.

**Reply by SpiderNet-Villach** [reply]
Thanks for the reply! I kind of fixed is this myself, but this is still a mess. Apparently, the Filebeat-Container can ping the Logstash-Container and vice versa (and that is why the "test output" command shows it would work) but the Filebeat-Daemon does not have connectivity at all. I tried switching the IPs, so the Filebeat-Container would talk to the Logstash-Container directly and that worked. However, this kills the SSL connection.  

Nevertheless, this is pretty strange - Shouldn't Filebeat and Logstash work out of the box in Security Onion 2.3?

**Reply by weslambert** [reply]
Yes, you shouldn't need any additional changes or configuration.
